app/views/shared/_nav.html.haml

```diff
@@ -12,3 +11,19 @@
     -else
       = menu_item "Sign In", '/auth/artsy/new'
 = bootstrap_flash
+
+- unless request.path.include?('/v1')
+  .alert.alert-warning
+    %p
+      Please note, we are in the process of retiring the
+      = link_to 'public api.', '/v2'
+      The public api, as well as its documentation and playground, will remain available until
+      %strong June 30th, 2025.
+    %p
+      This change will not affect partners currently using our
+      = link_to 'partner api.', '/v1'
+    %p
+      If you're interested in integrating your service with Artsy's partner api, please reach out to ....
```

Comment:
@ansor4 Do we want to put a contact here, or drop this?

Reply:
ooo, good q, we should put an email there I just don't know which one. Maybe we leave this line out until we figure it out?